.gitignore

```diff
@@ -2,7 +2,7 @@
 # https://github.com/netlify/zip-it-and-ship-it/issues/38
```

Comment:
Even though the issue is fixed, I imagine we still want to keep these around since developers might have the file in their local environments?

Reply:
Yeah that was my thinking, leave it for now